.github/workflows/main.yml

```diff
@@ -1,23 +1,19 @@
-name: Ruby
+name: CI
```

Comment:
I renamed this from Ruby, because it felt redudant having `Ruby` then `Ruby 3.0.4`.